secrets/.env_template

```diff
@@ -0,0 +1,5 @@
+# Fill this .env_template with the values required and rename it .env
+
+# POSTGRES_DB=tempaqua_sample
```

Comment:
DB : tempaqua
user : tempaqua_admin
schema : tempaqua_sample

Open to discuss @nicdub